File: CPP/Kode Program/Baju.cpp
#include <iostream>
#include <string>
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris
{
    private:
        string Untuk;
        string Size;
        string Merk;

    public:
        // Konstruktor
        Baju() 
        {

        }

        // Constructor dengan Parameter
        Baju(string id, string nama, string kategori, int harga, string jenis, string bahan, string warna, string untuk, string size, string merk)
        {
            setId(id);
            setNamaProduk(nama);
            setKategoriProduk(kategori);
            setHargaProduk(harga);
            setJenis(jenis);
            setBahan(bahan);
            setWarna(warna);
            setUntuk(untuk);
            setSize(size);
            setMerk(merk);
        }

        // Setter dan Getter Untuk
        void setUntuk(string untuk)
        {
            Untuk = untuk;
        }

        string getUntuk()
        {
            return Untuk;
        }

        // Setter dan Getter Size
        void setSize(string size)
        {
            Size = size;
        }

        string getSize()
        {
            return Size;
        }

        // Setter dan Getter Merk
        void setMerk(string merk)
        {
            Merk = merk;
        }

        string getMerk()
        {
            return Merk;
        }

        // Destruktor
        ~Baju() 
        {
            
        }
};

File: Java/Kode Program/Baju.java

class Baju extends Aksesoris
{
    
    private String untuk;
    private String size;
    private String merk;

    public Baju() 
    {
        // Konstruktor default kosong
    }

    public Baju(String id, String nama, String kategori, int harga, String jenis, String bahan, String warna, String untuk, String size, String merk) 
    {
        setId(id);
        setNamaProduk(nama);
        setKategoriProduk(kategori);
        setHargaProduk(harga);
        setJenis(jenis);
        setBahan(bahan);
        setWarna(warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public void setUntuk(String untuk) 
    {
        this.untuk = untuk;
    }

    public String getUntuk() 
    {
        return untuk;
    }

    public void setSize(String size) 
    {
        this.size = size;
    }

    public String getSize() 
    {
        return size;
    }

    public void setMerk(String merk) 
    {
        this.merk = merk;
    }

    public String getMerk() 
    {
        return merk;
    }
}
